ios/FluentUI/Shimmer/ShimmerView.swift

```diff
@@ -5,6 +5,35 @@
 
 import UIKit
 
+/// Shimmer style can be either concealing or revealing
+/// The style affects the default shimmer alpha value and the default shimmer tint color
+@objc(MSFShimmerViewStyle)
+public enum ShimmerStyle: Int, CaseIterable {
+    /// Concealing shimmer: the gradient conceals parts of the subviews as it moves leaving most parts of the subviews unblocked.
+    case concealing
+
+    /// Revealing shimmer: the gradient reveals parts of the subviews as it moves leaving most parts of the subview blocked.
```

Comment:
It seems like the revealing style is more challenging since it has to match the background of the view that is being shimmed. It makes me think that the default tint color should be a property that can be set on the control.

But could also need to support scenarios where the background is a gradient like in OfficeMobile or even image backgrounds.

Do you think we could make a transparent gradient mask and animate it?
I believe it'd be the approach that supports most of the cases.

Reply:
I think you are referring to concealing shimmer? The current design of concealing shimmer actually does show most of the image by default (regardless of light or dark mode) with transparent gradient layers and a default gradient center color white. The user can set a different color with viewTintColor property as well to best match their images.